docs/dev_guide/how_to/workspace_setup.md

```diff
@@ -7,16 +7,31 @@ permalink: /docs/dev_guide/how_to/workspace_setup
 ---
 
 # Venice Workspace Setup
+We recommend using a Unix-based environment for development, such as Linux or macOS.
+If you're on Windows, we recommend using [WSL2](https://learn.microsoft.com/en-us/windows/wsl/install).
 
-## Checking out the code
-Venice's source code is hosted on [GitHub](https://www.github.com/linkedin/venice). Check out the code using:
+## Fork the Venice Repository
+Fork the Venice repo at https://github.com/linkedin/venice
+
+## Setting up the repository locally
 ```shell
-git clone https://github.com/linkedin/venice.git
+git clone https://github.com/${githubUsername}/venice.git
+cd venice
+git remote add upstream https://github.com/linkedin/venice.git
+git fetch upstream
 ```
 
+## Setting up Java
+We use Java 17 for development. You can download it [here](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html).
```

Comment:
nit: I personally still use 11 and we have been using 17 in production but i don't know if that's every component of venice or just venice-server.

Reply:
Feel free to drop this comment